.github/workflows/tests.yml

```diff
@@ -26,19 +26,18 @@ on:
 jobs:
   Tests:
     runs-on: ubuntu-20.04
-    timeout-minutes: 20
     strategy:
+      fail-fast: false
       matrix:
-          python-version: [3.7, 3.8, 3.9]
-          requirements-level: [pypi]
-          cache-service: [redis]
-          db-service: [postgresql10, postgresql13, mysql8]
+        python-version: ["3.9", "3.10", "3.11", "3.12"]
+        requirements-level: [pypi]
```

Comment:
Not needed anymore: we want to test only the latest (as you have correctly done below ☺️).
```suggestion

```